### PR TITLE
Add recipe for CRAN package RSclient

### PR DESCRIPTION
**Pull request by comatrion**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by comatrion** [comment]
Used https://github.com/bgruening/conda_r_skeleton_helper

**Comment by comatrion** [comment]
ref https://github.com/conda-forge/staged-recipes/pull/8653#issuecomment-506293966

**Comment by comatrion** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/r-rsclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - {{ native }}gcc-libs         # [win]
+
+test:
+  commands:
+    - $R -e "library('RSclient')"           # [not win]
+    - "\"%R%\" -e \"library('RSclient')\""  # [win]
+
+about:
+  home: http://www.rforge.net/RSclient/
+  license: GPL-2.0-or-later
```
Comment (cbrueffer) [inline recipes/r-rsclient/meta.yaml:48]:
```suggestion
  license: GPL-2.0-only
```

Comment (cbrueffer) [reply recipes/r-rsclient/meta.yaml:48]: Looks good apart from the license fix above.

**Comment by comatrion** [comment]
Hmm... a fail on windows build after changing only the license file.

**Comment by cbrueffer** [comment]
Please re-apply the fix, the build error was caused by an HTTP timeout. Happens all the time and only needs retriggering the build.

**Comment by comatrion** [comment]
Thank you!

**Pull request merged**